setup/Makefile

```diff
@@ -16,16 +21,34 @@ download-terraform: build-directories
 		unzip terraform_$(TERRAFORM_VERSION)_$$os.zip ; \
 		rm terraform_$(TERRAFORM_VERSION)_$$os.zip ; \
 		cd - ; \
-        done
+  done
 
 build: download-terraform
 	CGO_ENABLED=0 GOOS=linux   GOARCH=amd64 go build -ldflags "-s -X main.Version=$(L0_VERSION)" -a -o build/linux/l0-setup .
 	CGO_ENABLED=0 GOOS=darwin  GOARCH=amd64 go build -ldflags "-s -X main.Version=$(L0_VERSION)" -a -o build/darwin/l0-setup .
 	CGO_ENABLED=0 GOOS=windows GOARCH=amd64 go build -ldflags "-s -X main.Version=$(L0_VERSION)" -a -o build/windows/l0-setup.exe .
 
+install-smoketest: deps
+		for os in linux; do \
+                mkdir -p terra ; \
+                cd terra ; \
+                aws s3 cp s3://xfra-layer0/terraform/$$os/terraform_$(TERRAFORM_VERSION)_$$os.zip . ; \
+                unzip terraform_$(TERRAFORM_VERSION)_$$os.zip ; \
+                rm terraform_$(TERRAFORM_VERSION)_$$os.zip ; \
+                cd - ; \
+                cp ./terra/terraform ~/bin/ ; \
+    done
+	  CGO_ENABLED=0 GOOS=linux   GOARCH=amd64 go build -ldflags "-s -X main.Version=$(L0_VERSION)" -a -o ./l0-setup .
```

Comment:
could work with `cli`, but I had problems w `l0-setup` and its runtime context. When building the `l0-setup` binary, it needed to be run alongside the files in `/setup`. Maybe you have a trick to get around that.